### PR TITLE
Find and replace all _([:alpha:][:alnum:]*) with _{\mathrm{$1}}

### PR DESCRIPTION
**Pull request by jamesray1**
... using Libre Open Office with regular expressions ticked.

### CODE REVIEW

**Comment by jamesray1** [comment]
For 87edbc6, I meant "Remove `mathrm`s in a URL.

**Comment by jamesray1** [comment]
@pirapira, please merge this first to avoid further conflicts that will be time-consuming to resolve.

**Comment by pirapira** [comment]
That makes sense.

File: Paper.tex [hunk 1]
```diff
@@ -116,7 +116,7 @@ \section{The Blockchain Paradigm} \label{ch:overview}
 
 Ethereum, taken as a whole, can be viewed as a transaction-based state machine: we begin with a genesis state and incrementally execute transactions to morph it into some final state. It is this final state which we accept as the canonical ``version'' of the world of Ethereum. The state can include such information as account balances, reputations, trust arrangements, data pertaining to information of the physical world; in short, anything that can currently be represented by a computer is admissible. Transactions thus represent a valid arc between two states; the `valid' part is important---there exist far more invalid state changes than valid state changes. Invalid state changes might, \eg, be things such as reducing an account balance without an equal and opposite increase elsewhere. A valid state transition is one which comes about through a transaction. Formally: 
 \begin{equation}
-\linkdest{Upsilon_state_transition}\boldsymbol{\sigma}_{t+1} \equiv \Upsilon(\boldsymbol{\sigma}_t, T)
+\linkdest{Upsilon_state_transition}\boldsymbol{\sigma}_{t+1} \equiv \Upsilon(\boldsymbol{\sigma}_{\mathrm{t}}, T)
```
Comment (pirapira) [inline Paper.tex]: `t` is a variable here.  So it should stay italic.

File: Paper.tex [hunk 2]
```diff
@@ -127,7 +127,7 @@ \section{The Blockchain Paradigm} \label{ch:overview}
 
 Formally, we expand to:
 \begin{eqnarray}
-\boldsymbol{\sigma}_{t+1} & \equiv & \Pi(\boldsymbol{\sigma}_t, B) \\
+\boldsymbol{\sigma}_{t+1} & \equiv & \Pi(\boldsymbol{\sigma}_{\mathrm{t}}, B) \\
```
Comment (pirapira) [inline Paper.tex]: Here as well, `t` is a variable and should stay italic.

**Comment by jamesray1** [comment]
@pirapira, it's fixed, again, please merge this first.

**Comment by pirapira** [comment]
Oh, I missed your message.  I'm fixing the conflicts myself now.

File: Paper.tex [hunk 3]
```diff
@@ -210,20 +210,20 @@ \subsection{World State} \label{ch:state}
 The account state comprises the following four fields:
 
 \begin{description}
-\item[nonce] \linkdest{account_nonce}A scalar value equal to the number of transactions sent from this address or, in the case of accounts with associated code, the number of contract-creations made by this account. For account of address $a$ in state $\boldsymbol{\sigma}$, this would be formally denoted $\boldsymbol{\sigma}[a]_n$.
```
Comment (pirapira) [inline Paper.tex]: Wait, wait, the change on account_nonce is weird.

File: Paper.tex [hunk 4]
```diff
@@ -326,68 +326,67 @@ \subsection{The Block} \label{ch:block}
...
-\item[gasUsed]\linkdest{block_gas_used_H__g}{} A scalar value equal to the total gas used in transactions in this block; formally $H_g$.
-\item[timestamp]\linkdest{block_timestamp_H__s}{} A scalar value equal to the reasonable output of Unix's time() at this block's inception; formally $H_s$.
-\item[extraData]\linkdest{block_extraData_H__x}{} An arbitrary byte array containing data relevant to this block. This must be 32 bytes or fewer; formally $H_x$.
-\item[mixHash]\linkdest{mixHash_H__m}{} A 256-bit hash which, combined with the nonce, proves that a sufficient amount of computation has been carried out on this block; formally $H_m$.
-\item[nonce]\linkdest{block_nonce_H__n}{} A 64-bit hash which, combined with the mix-hash, proves that a sufficient amount of computation has been carried out on this block; formally $H_n$.
+\item[parentHash]\linkdest{parent_Hash_H__p_def_words}{} The Keccak 256-bit hash of the parent block's header, in its entirety; formally $H_{\mathrm{p}}$.
+\item[ommersHash]\linkdest{beneficiary_H__c}{} The Keccak 256-bit hash of the ommers list portion of this block; formally $H_{\mathrm{o}}$.
```
Comment (pirapira) [inline Paper.tex]: Why does the linkdest move?

**Comment by pirapira** [comment]
Tested locally.

**Pull request merged**